checkov/github/dal.py

```diff
@@ -72,7 +73,7 @@ def get_organization_security(self):
                         }
                     }
                 }
-                """, variables={'org': 'bridgecrewio'})
+                """, variables={'org': self.org})
```

Comment:
does anything bad happen, when this is an empty string?

Reply:
Nope, it just doesn't fetch any data